web-admin/src/components/projects/project-status-store.ts

```diff
@@ -0,0 +1,176 @@
+import type { V1DeploymentStatus } from "@rilldata/web-admin/client";
+import type {
+  RpcStatus,
+  V1GetProjectResponse,
+} from "@rilldata/web-admin/client";
+import { getDashboardsForProject } from "@rilldata/web-admin/components/projects/dashboards";
+import { invalidateProject } from "@rilldata/web-admin/components/projects/invalidations";
+import type {
+  QueryKey,
+  QueryClient,
+  CreateQueryResult,
+} from "@tanstack/svelte-query";
+import { get, Readable, Writable, writable } from "svelte/store";
+
+export type ProjectStatusState = {
```

Comment:
Overall this file seems like a bunch of work to duplicate what we get in response from 
```ts
const proj = createAdminServiceGetProject(organization, project)
```

I don't see the value-add. Can't you just use the Tanstack store (`$proj`) that has all this info?

Reply:
Wasnt aware you can just pass a method for refetch interval. This can be simplified by making it a wrapper createQuery function